README.md

```diff
@@ -12,7 +12,9 @@ Binoc is an automatic software maintainer in a box. It builds off of other proje
 (Others Comming Soon)
 
 ## Usage
-Binoc can be run as either a GitHub Action or a Docker Container
+Binoc can be run as either a GitHub Action or a Docker Container. You can have binoc
+manage your pull requests (the default) or set `skip_pr` to "true" to manage them on
```

Comment:
I'm super left-handedly dyslexic so I've done this a billion times but I think `skip_pr` is supposed to be `pr_skip`?